components/cam/src/physics/cam/micro_p3.F90

```diff
@@ -1149,6 +1143,14 @@ SUBROUTINE p3_main(qc,nc,qr,nr,th,qv,dt,qitot,qirim,nitot,birim,   &
     !----- Local variables and parameters:  -------------------------------------------------!
     !
 
+    ! These outputs are no longer provided by p3_main.
```

Comment:
Why not?

Reply:
They were added in the first round of p3 development because they are present in MG2 and we were using that as a template.  Subsequent code clean-up tasks have removed their use in the post-processing so they are no longer needed explicitly from P3.